test/metabase/upload_test.clj

```diff
@@ -509,7 +509,10 @@
                 "\" 3\";;           b;false;\"$ 1,000.1\";2022-02-01;2022-02-01T00:00:00"]
    :tab        ["id    \tnulls\tstring \tbool \tnumber       \tdate      \tdatetime"
                 "2   \t\t          a \ttrue \t1.1        \t2022-01-01\t2022-01-01T00:00:00"
-                "\" 3\"\t\t           b\tfalse\t\"$ 1,000.1\"\t2022-02-01\t2022-02-01T00:00:00"]})
+                "\" 3\"\t\t           b\tfalse\t\"$ 1,000.1\"\t2022-02-01\t2022-02-01T00:00:00"]
+   :pipe       ["id    |nulls|string |bool |number       |date      |datetime"
+                "2\t   ||          a |true |1.1\t        |2022-01-01|2022-01-01T00:00:00"
+                "\" 3\"||           b|false|\"$ 1,000.1\"|2022-02-01|2022-02-01T00:00:00"]})
```

Comment:
Do we need to worry about quoted non-separating pipes?

Reply:
Ah yeah, that would have been nice. Not sure it's worth another round though, and that logic is tested in our dependency.